README.md

```diff
@@ -5,13 +5,25 @@
 
 The Yellow Paper is a formal definition of the Ethereum protocol, originally by Gavin Wood, currently maintained by Nick Savers and with contributions from many people around the world.
 
-It is free culture work, licensed under the Creative Commons Attribution Share-Alike (CC-BY-SA) version 4.0.
+It is a free culture work, licensed under Creative Commons Attribution Share-Alike (CC-BY-SA) version 4.0.
 
 ## Usage
 
-The paper comes as a single ``latex`` file ``Paper.tex``. The latest version is generally available as a PDF at https://ethereum.github.io/yellowpaper/paper.pdf or just [yellowpaper.io](http://yellowpaper.io/) .
+The latest version is generally available as a PDF at https://ethereum.github.io/yellowpaper/paper.pdf. With the hyperref package, this version of the yellow paper can be used to get links that direct to specific info in the yellow paper, which is useful e.g. for pedagogical and academic purposes.
```

Comment:
The new sentence is not descriptive enough.  After reading it, I don't know how to obtain the link URL.

Reply:
I was going to add a link to the file that is added with https://github.com/ethereum/yellowpaper/pull/433. "For further info, see [here](<insert URL to Using-the-hyperref-package.md>).